packages/opal-common/opal_common/utils.py

```diff
@@ -101,7 +104,9 @@ def getSignatureKey(key, dateStamp, regionName, serviceName):
         + payload_hash
     )
 
-    region = "us-east-1"
+    if not region:
+        region = "us-east-1"
```

Comment:
For clarity, I would make this the default for `POLICY_BUNDLE_SERVER_AWS_REGION` in `opal_server/config.py`

Reply:
Will update default value, and remove this statement.